app/controllers/mixins/ems_common_angular.rb

```diff
@@ -121,7 +121,7 @@ def get_task_args(ems)
       when 'ManageIQ::Providers::Azure::CloudManager'
         [user, password, params[:azure_tenant_id], params[:subscription], nil, params[:provider_region]]
       when 'ManageIQ::Providers::Vmware::CloudManager'
-        [params[:default_hostname], params[:default_api_port], user, password, true]
+        [params[:default_hostname], params[:default_api_port], user, password, params[:api_version], true]
```

Comment:
Is `:api_version` available in `params` at this point?  Do we need to set a `:default_api_version` of `5.5`?

Reply:
@agrare I put debug point in here once again and yes, `params[:api_version]` is available and reflects whatever user picked in the dropdown. I don't think we need to deal with default values here since (a) dropdown doesn't offer empty value and (b) even if it would and `nil` would be passed, we handle the default on vCloud itself [link](https://github.com/ManageIQ/manageiq-providers-vmware/pull/219/files#diff-c1348e23e033162d75ce8f1a6a5d4081R41).

But thanks for pointing out, I wasn't sure either when preparing the PR, so I had to do it "the hard way" with debugger.

Reply:
Ah okay I didn't realize the user had to select a version